gldemo/plugin.cpp

```diff
@@ -321,8 +366,11 @@ class gldemo : public threadloop {
 		createSharedEyebuffer(&(eyeTextures[0]));
 		createSharedEyebuffer(&(eyeTextures[1]));
 
+		createSharedDepthBuffer(&(eyeDepthTextures[0]));
+		createSharedDepthBuffer(&(eyeDepthTextures[1]));
+
 		// Initialize FBO and depth targets, attaching to the frame handle
-		createFBO(&(eyeTextures[0]), &eyeTextureFBO, &eyeTextureDepthTarget);
+		createFBO(&(eyeTextures[0]), &eyeTextureFBO, &(eyeDepthTextures[0]), &eyeTextureDepthTarget);
```

Comment:
There is a gap in my understanding of OpenGL here. Why create the FBO with just the 0th textures? How does the right eye (1st index) work?